bindings/wasm/src/did/wasm_document.rs

```diff
@@ -496,4 +675,25 @@ impl From<IotaDocument> for WasmDocument {
 extern "C" {
   #[wasm_bindgen(typescript_type = "DIDUrl | string")]
   pub type UDIDUrlQuery;
+
+  #[wasm_bindgen(typescript_type = "string | string[] | null")]
+  pub type UOneOrManyUrl;
+
+  #[wasm_bindgen(typescript_type = "DID | DID[] | null")]
+  pub type UOneOrManyDID;
+
+  #[wasm_bindgen(typescript_type = "DID[]")]
+  pub type ArrayDID;
+
+  #[wasm_bindgen(typescript_type = "Service[]")]
+  pub type ArrayService;
+
+  #[wasm_bindgen(typescript_type = "Array<string>")]
+  pub type ArrayString;
+
+  #[wasm_bindgen(typescript_type = "VerificationMethod[]")]
+  pub type ArrayMethods;
```

Comment:
Nit: follow naming conventions.

```suggestion
  #[wasm_bindgen(typescript_type = "VerificationMethod[]")]
  pub type ArrayVerificationMethod;
```